Resources/views/Form/stylesheet_layout.html.twig

```diff
@@ -1,7 +1,9 @@
 {% block form_stylesheet %}
 {% spaceless %}
     {% for child in form %}
-        {{ form_stylesheet(child) }}
+        {% if child.vars.pickerOptions is defined %}
```

Comment:
What I mean is that with this change the needed CSS files will never be loaded in case the form type provided by this bundle is nested in the form type tree as the block doesn't descend the structure.